extensions/script-libraries/dropbox/dropbox.livecodescript

```diff
@@ -0,0 +1,5729 @@
+
+pCallback: The handler to call when the request is complete. If empty the command
+will block until complete. Requires tsNet with LiveCode Indy License.
+
+The result: An error if the HTTP request failed
+
+It: For asynchronous requests it will contain a request ID. This may be used with
+tsNet's tsNetGetStatus function. If the request is synchronous and successful the it
+variable will be a UFT8 encoded JSON string in the form:
+
+````
+{
+".tag": "file",
+"name": "Prime_Numbers.txt",
+"id": "id:a4ayc_80_OEAAAAAAAAAXw",
+"client_modified": "2015-05-12T15:50:38Z",
+"server_modified": "2015-05-12T15:50:38Z",
+"rev": "a1c10ce0dd78",
+"size": 7212,
+"path_lower": "/homework/math/prime_numbers.txt",
+"path_display": "/Homework/math/Prime_Numbers.txt",
+"sharing_info": {
+"read_only": true,
+"parent_shared_folder_id": "84528192421",
+"modified_by": "dbid:AAH4f99T0taONIb-OurWxbNQ6ywGRopQngc"
+},
+"property_groups": [
+{
+"template_id": "ptid:1a5n2i6d3OYEAAAAAAAAAYa",
+"fields": [
+{
+"name": "Security Policy",
+"value": "Confidential"
+}
+]
+}
+],
+"has_explicit_shared_members": false
+}
+````
+
+If an error occurred the JSON string will be in the following form:
+
+````
+{
+"error_summary": string,
+"error": {
+".tag": string
+}
+}
+````
+*/
+command dropboxMove pAccessToken, pPathFrom, pPathTo, pCallback
+   local tHeader, tPost, tURL
+   put __move_HEADER(pAccessToken) into tHeader
+   put __move_POST(pPathFrom,pPathTo) into tPost
+   put __move_URL() into tUrl
+   
+   __API tHeader, tPost, tURL, pCallback
+   if the result is empty then
+      return it for value
+   else
+      return the result for error
+   end if
+end dropboxMove
+
+/**
+
+Permanently delete the file or folder at a given path.
+
+Syntax: dropboxPermanetlyDelete <pAccessToken>, <pPath>, [<pCallback>]
+
+Description:
+> *Note:* This endpoint is only available for Dropbox Indy apps.
+
+If the callback parameter is not empty the request will be asynchronus and
+when complete the callback will be sent to the object that accessed the API.
+The callback will be sent with three parameters:
+
+- The request ID which will be the value of the it variable after calling the command
+- The HTTP response code
+- The data returned which will be the same data as documented for the it variable in a synchronous request.
+
+
+Parameters:
+pAccessToken: An OAuth2 Access token to access the user's account
+
+pPath: Path in the user's Dropbox to be deleted.
+
+pCallback: The handler to call when the request is complete. If empty the command
+will block until complete. Requires tsNet with LiveCode Indy License.
+
+The result: An error if the HTTP request failed
+
+It: For asynchronous requests it will contain a request ID. This may be used with
+tsNet's tsNetGetStatus function.
+
+If an error occurred the JSON string will be in the following form:
+
+````
+{
+"error_summary": string,
+"error": {
+".tag": string
+}
+}
+````
+*/
+command dropboxPermanetlyDelete pAccessToken, pPath, pCallback
```

Comment:
I think this command is supposed to be `dropboxPermanentlyDelete` -- there's a missing "n" in "permanently".